go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/corona10/goimagehash
 
-require github.com/nfnt/resize v0.0.0-20160724205520-891127d8d1b5
+go 1.13
```

Comment:
Please remove this line. :)

Reply:
Sorry, I saw the message, but I forgot about it...
Busy days :)

You don't want to pin the version of `nfnt/resize`, is that it?
Or are you talking about the `go 1.13` line and select it wrong here?

Reply:
I mean go 1.13 :)

Reply:
I discussed about this issue before with another contributor.
Please refer this thread :)
https://github.com/corona10/goimagehash/pull/34#discussion_r349050336

Reply:
Sure, but is there any specific reason for that? If is, I'm not aware of it and I would like to learn about.

Reply:
I have removed the line. Interesting topic, to be honest I was curious about that directive since they introduced it on 1.12.
I knew for sure that this was not the minimum required version of the toolchain.

For now, I didn't fully understand the behavior of that directive on the build process to be honest, there is a lot of answer on the internet and everyone is saying something different.
I will run some tests (because I'm curious) and then decide if I will use it on my projects or not.

Thanks for pointing out the discussion.